packages/typespec-ts/package.json

```diff
@@ -33,7 +33,7 @@
     "@types/fs-extra": "^9.0.13",
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
-    "@typespec/http": "~0.44.0",
+    "@typespec/http": ">=0.45.0 <1.0.0",
```

Comment:
I should leave the comment here

Reply:
it's not removing

Reply:
I removed already, I think it's out of date.